packages/flutter/test/widgets/reorderable_list_test.dart

```diff
@@ -1483,6 +1483,51 @@ void main() {
       await testMove(element.$1, element.$2, reverse: true, scrollDirection: Axis.horizontal);
     }
   });
+
+  testWidgets('Tests that the item position is correct when prototypeItem or itemExtent are set', (WidgetTester tester) async {
+    Future<void> pumpFor({Widget? prototypeItem, double? itemExtent}) async {
+      await tester.pumpWidget(
+        MaterialApp(
+          home: Scaffold(
+            body: CustomScrollView(
+              slivers: <Widget>[
+                SliverReorderableList(
+                  itemBuilder: (BuildContext context, int index) {
+                    return ReorderableDragStartListener(
+                      key: ValueKey<int>(index),
+                      index: index,
+                      child: SizedBox(
+                        height: 100,
+                        child: Text('$index'),
+                      ),
+                    );
+                  },
+                  itemCount: 5,
+                  itemExtent: itemExtent,
+                  prototypeItem: prototypeItem,
```

Comment:
Hmm, this looks like a bug. You should not be able to use itemExtent and prototypeItem together.

Which one does it end up using?

We assert against this in other ScrollViews, like ListView.

Reply:
They will not exist at the same time. This is just to provide a convenient method for testing `prototypeItem` and `itemExtent` respectively.